### PR TITLE
Add option to create non-ha project.yml template [skip ci]

### PR DESCRIPTION
**Pull request by IsaacYangSLA**

### CODE REVIEW

File: docs/programming_guide/provisioning_system.rst [hunk 1]
```diff
@@ -397,13 +397,21 @@ Edit the project.yml configuration file to meet your project requirements:
 Default project.yml file
 ========================
 
-The following is an example of the default project.yml file.
+The following is an example of the default project.yml file of HA mode.
 
-.. literalinclude:: ../../nvflare/lighter/project.yml
+.. literalinclude:: ../../nvflare/lighter/hd_project.yml
```
Comment (nvkevlu) [inline docs/programming_guide/provisioning_system.rst:402]: Is hd_project.yml supposed to be ha_project.yml?

Comment (IsaacYangSLA) [reply docs/programming_guide/provisioning_system.rst:402]: Good catch.  Thanks.  Let me fix it.

**Pull request merged**